internal/verifier/generation.go

```diff
@@ -16,6 +16,22 @@ const (
 	generationFieldName = "generation"
 )
 
+type generationDoc struct {
+	Generation      int `bson:"generation"`
+	MetadataVersion int
```

Comment:
Does this not need a `bson` tag? If it doesn't, then why does `Generation` need it?

Reply:
Neither of these probably need the tag. `Generation` has it, I think, just because it’s always had it.

Reply:
Removed after confirming that it make no difference as written.